src/utils/config/project.py

```diff
@@ -40,15 +42,24 @@ def config_path(self) -> Path:
 
     @property
     def ordered_dict(self):
+        self._ensure_config_loaded()
         general = OrderedDict(**self.config.__dict__)
         general.pop("contracts")
 
         result = OrderedDict()
         result["protostar.general"] = general
-        assert self.config, "No package configuration loaded!"
         result["protostar.contracts"] = self.config.contracts
         return result
 
+    def get_include_paths(self) -> List[str]:
+        self._ensure_config_loaded()
+        libs_path = Path(self.project_root, self.config.libs_path)
+        return [
+            str(self.project_root),
+            str(libs_path),
+            *collect_immediate_subdirectories(libs_path),
```

Comment:
Why is `collect_immediate_subdirectories` needed?

Reply:
Because when we're loading sources from libraries, the libraries refer to the source files from the root of the library. See openzeppelin for example - they import straight from openzeppelin which is not in our `cairo_path` if we don't include `cairo_contracts` in `cairo_path`